### PR TITLE
ANMN acoustic pipeline to uppdate deployment metadata

### PR DESCRIPTION
**Pull request by mhidas**

### CODE REVIEW

**Comment by mhidas** [comment]
Requires https://github.com/aodn/chef-private/pull/2008

**Comment by mhidas** [comment]
This is ready to be merged. @ggalibert Would you mind having a quick look?

File: watch.d/ANMN_PA.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "path": [ "ANMN/Acoustic" ],
+  "execute": "ANMN/acoustic/incoming_handler.sh"
+}
```
Comment (ggalibert) [inline watch.d/ANMN_PA.json]: Is the casing inconsistency OK with "A/acoustic" either in path or execute?

**Comment by ggalibert** [comment]
Apart from that one comment above, the rest sounds good to me.

File: watch.d/ANMN_PA.json (hunk 1 above)
Comment (mhidas) [inline watch.d/ANMN_PA.json]: Yeah, it's ok. Those are currently the correct paths. Maybe one day I'll rename the code directory (i.e. in this repo) to make it consistent, but I don't think it's a big deal.

**Pull request merged**